### PR TITLE
Adding support for negative indices.

### PR DESCRIPTION
**Pull request by LucianoPAlmeida**
Found this pretty cool :) 
Try to Implementing #1  Let me know what you think :)

### CODE REVIEW

**Comment by NSMyself** [comment]
@LucianoPAlmeida damn, I had just finished a fork for the exact same thing and was about to open a PR 😛.

Nicely done, I prefer your implementation to my own actually, quite clean 👍 

By the way, I don't know the author's opinion but I think that adding support to a negative index just to get a specific character would make sense (eg: `let lastChar = "abcdefg"[-1]`). 
It'd be pretty trivial stuff considering the remaining methods are already implemented.

File: Tests/StryngTests/StryngTests.swift [hunk 1]
```diff
@@ -42,6 +42,11 @@ class StryngTests: XCTestCase {
         let example = "Example"
         XCTAssertNil(example[0...8])
         XCTAssertNil(example[10...11])
+
```
Comment (BalestraPatrick) [inline Tests/StryngTests/StryngTests.swift]: Can you please split into separate tests these with negative indices?

**Review by BalestraPatrick — CHANGES_REQUESTED** [review]

File: Tests/StryngTests/StryngTests.swift (hunk 1 above)
Comment (LucianoPAlmeida) [reply Tests/StryngTests/StryngTests.swift]: Sure, Done :)

**Comment by BalestraPatrick** [comment]
Thank you @LucianoPAlmeida! I am deploying a new release tonight 🎉

**Pull request merged**